src/set.ts

```diff
@@ -217,4 +217,23 @@ export class SuperchargedSet<T> implements Iterable<T> {
   toArray (): T[] {
     return Array.from(this.set)
   }
+
+  /**
+   * Appends values to the end of the array.
+   *
+   * @param {*} values
+   *
+   * @returns {SuperchargedSet}
+   */
+  concat (...values: Array<T | T[]>): this {
+    for (const value of values) {

```

Comment:
The set already has a `.toArray()` method creating an array of the set’s entries. You may then combine it with the array’s concat method, like `this.toArray().concat(...values)`. What do you think?

Reply:
I like the `.toArray()` way and have updated the code.